src/models/history.ts

```diff
@@ -585,6 +586,33 @@ export namespace History {
     return getFinishedSets(entry).length;
   }
 
+  export function getRecordText(record: IHistoryRecord, settings: ISettings): string {
+    // TODO: This isn't actually filtering to the completed sets
+    const entries = record.entries.filter((e) => e.sets.filter((s) => (s.completedReps ?? 0) > 0).length > 0);
```

Comment:
@astashov I thought this which I found elsewhere in the code would get me to a set of sets (ha) that are completed, but when testing I'm still getting all the initial sets from the entries, not what was actually done.